notebooks/SimpleExample.ipynb

```diff
@@ -16,19 +16,7 @@
    "cell_type": "code",
```

Comment:
<p>wow this is wonderful</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ctsm_py/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>